Plugins/MeshFeatureTracking/src/MeshFeatureTrackingPlugin.cpp

```diff
@@ -51,6 +48,8 @@ bool MeshFeatureTrackingPluginC::doAddWidget( QString& name ) {
 }
 
 QWidget* MeshFeatureTrackingPluginC::getWidget() {
```

Comment:
Are we shure that getWidget is called before registerPlugin ? if not there will be a connect to a nullptr m_widget.

Reply:
the connection shoud be made right after creation of the widget, i.e. in `getWidget`

Reply:
@hoshiryu you commented an outdated change.
Is there anything that should be updated in the current version ?